src/App.js

```diff
@@ -4,9 +4,61 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      Hello world!
+      <MessageElement></MessageElement>
```

Comment:
component without children please use shortcut for component
`<MessageElement />` wherever needed